src/main/scala/analysis/MemoryModelMap.scala

```diff
@@ -22,7 +22,7 @@ class MemoryModelMap {
   private val MAX_BIGINT: BigInt = BigInt(Long.MaxValue)
   private val contextStack = mutable.Stack.empty[String]
   private val sharedContextStack = mutable.Stack.empty[List[StackRegion]]
-  private val localStacks = mutable.Map[String, List[StackRegion]]()
+  private val localStacks = mutable.Map[String, List[StackRegion]]().withDefaultValue(List.empty)
```

Comment:
It's really best to avoid using withDefaultValue where possible since it has some strange properties that can cause unintuitive bugs.